entrypoint-node.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+mkdir -p /dev/net
```

Comment:
I have tested it, it didn't works. Sorry, I know less about mknod. entrypoint way is not a hack, I would recommend it. Later you can preheat some other stuff in this script.